frontend/src/app/user/user.component.ts

```diff
@@ -27,20 +28,26 @@ export class UserComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    this.loginForm = this.userFormService.createForm(this.user);
+    this.userForm = this.userFormService.createForm(this.user);
   }
 
   onClickRegister() {
-    FormUtil.markAsTouched(this.loginForm);  // macht Validierungsfehler sichtbar
-    if (this.loginForm.valid && !this.loginForm.pending) {  // Form ist gültig und die Validierung ist abgeschlossen
-      console.log('form value', this.loginForm.value);
+    FormUtil.markAsTouched(this.userForm);  // macht Validierungsfehler sichtbar
+    if (this.userForm.valid && !this.userForm.pending) {  // Form ist gültig und die Validierung ist abgeschlossen
+      console.log('form value', this.userForm.value);
       console.log('send data to Service');
-      const fb = this.loginForm.value;
-      this.user = this.userFormService.mergeUser(this.loginForm.value, this.user);
-      this.businessService.saveUser(this.user);
+      this.businessService.register(createRegisterRequest(this.userFormService.createUser())).subscribe( response => {
```

Comment:
Der Umbau auf createUser hat 2 Effekte:
- der user wird Redundant gehalten - in der Componente und im Service.
- die User Componente hat 2 Modi, im Register wird ein neuer User  erstellt. Im Mutate (user ist eingeloggt) wird ein user geändert, wird jetzt ein Create gemacht, gehen Daten die nicht im Form sind verloren, z.b. Id.

Reply:
ok, lasse das so wie es jetzt ist.